source/include/accel/data.hh

```diff
@@ -74,18 +74,18 @@ struct RuntimeOptions
         memcpy(grid_size.data(), _grid_size, 3 * sizeof(int));
         memcpy(block_size.data(), _block_size, 3 * sizeof(int));
 
-        if(device.key == "gpu")
+#if defined(TOMOPY_USE_OPENCV)
+        if(device.key == "cpu")
         {
-#if defined(TOMOPY_USE_CUDA)
-            interpolation = GetNppInterpolationMode(_interp);
-#else
             interpolation = GetOpenCVInterpolationMode(_interp);
-#endif
         }
-        else
+#endif
+#if defined(TOMOPY_USE_CUDA)
+        if(device.key == "gpu")
         {
-            interpolation = GetOpenCVInterpolationMode(_interp);
+            interpolation = GetNppInterpolationMode(_interp);
         }
+#endif
```

Comment:
This is probably the real cause of the segfault because the default value of `interpolation` is -1.

Reply:
Device key can no longer be 'gpu' or 'cpu' if the supporting library wasn't included.